definitions/Uno.def

```diff
@@ -1,58 +1,50 @@
 Bootstrap: docker
-From: tensorflow/tensorflow:2.11.0-gpu
+From: python:3.7
 
 %labels
-	MAINTAINERS	woz@anl.gov and weaverr@anl.gov
-	VERSION	v0.0.2
+        MAINTAINER      woz@anl.gov / weaverr@anl.gov
+        VERSION v0.0.2
 
 %setup
-	mkdir -p $SINGULARITY_ROOTFS/IMPROVE
-	cp src/Singularity_gpu_fix.sh $SINGULARITY_ROOTFS/IMPROVE/
+        mkdir -p $SINGULARITY_ROOTFS/IMPROVE
+        cp src/Singularity_gpu_fix.sh $SINGULARITY_ROOTFS/IMPROVE/
 
 %environment
-	# Available at runtime not build time
-	export MODEL="Uno"
-	
-	# WORK_DIR=$SINGULARITY_ROOTFS/IMPROVE/
-	
-    	export CANDLE_DATA_DIR=/candle_data_dir
-	export IMPROVE_DATA_DIR=/candle_data_dir
-	export PYTHONPATH=/usr/local/IMPROVE:$PYTHONPATH
-	export IMPROVE_MODEL_DIR=/usr/local/Benchmarks/Pilot1/Uno_IMPROVE/
+        # Available at runtime not build time
+        MODEL="Uno"
+        PATH=$PATH:$SINGULARITY_ROOTFS/IMPROVE/${MODEL}
+        WORK_DIR=$SINGULARITY_ROOTFS/IMPROVE/
+        CANDLE_DATA_DIR=/candle_data_dir
+        export PYTHONPATH=/opt/IMPROVE/:$PYTHONPATH
 
 %post
     # Install prerequisites and update image
     # See this thread for more information about apt-keys for NVIDIA:
     # https://forums.developer.nvidia.com/t/invalid-public-key-for-cuda-apt-repository/212901/18
-    apt-get install wget
+    # apt-get install wget
     # apt-key del 7fa2af80
     # rm -v /etc/apt/sources.list.d/nvidia-ml.list /etc/apt/sources.list.d/cuda.list
     # wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.0-1_all.deb
     # dpkg -i cuda-keyring_1.0-1_all.deb
     apt-get update
     apt-get install -y git ed
-    python -m pip install numba scikit-learn
+    python -m pip install tensorflow==2.10.0 scikit-learn==1.0.2 pyarrow==12.0.1 joblib==1.3.2
 
     mkdir -p $SINGULARITY_ROOTFS/IMPROVE /candle_data_dir
     cd $SINGULARITY_ROOTFS/IMPROVE
     chmod a+x Singularity_gpu_fix.sh
     ./Singularity_gpu_fix.sh
     cd /usr/local
-    git clone https://github.com/ECP-CANDLE/candle_lib.git
+    git clone -b develop https://github.com/JDACS4C-IMPROVE/IMPROVE.git /opt/IMPROVE
```

Comment:
We have to make IMPROVE installable via pip. Right now you have to modify PYTHONPATH.

Reply:
Yep that's my understanding of the current state of IMPROVE lib too